java/src/org/openqa/selenium/grid/commands/CompletionCommand.java

```diff
@@ -33,10 +33,6 @@
 import java.util.Set;
 import java.util.stream.Collectors;
 import java.util.stream.StreamSupport;
-import org.openqa.selenium.cli.CliCommand;
```

Comment:
Are you checking that this is compiling?

`CompletionCommand` extends `CliCommand`

Reply:
my bad, I've resolved conflicts incorrect

Reply:
To ensure that everything else is fine, I have executed [CI](https://github.com/iampopovich/selenium/actions/runs/4993603109) for the current branch to verify its stability and correctness.